build.gradle

```diff
@@ -459,60 +435,37 @@ dependencies {
         exclude group: "net.bytebuddy", module: "byte-buddy"
         exclude group: "net.bytebuddy", module: "byte-buddy-agent"
     }
-    testImplementation ('com.github.hmcts:rd-commons-lib:v0.0.14'){
-        exclude group: 'org.springframework.boot', module: 'spring-boot-starter-web'
-        exclude group: 'org.apache.groovy', module: 'groovy-json'
-        exclude group: 'org.apache.groovy', module: 'groovy'
-        exclude group: 'org.apache.groovy', module: 'groovy-xml'
-    }
-    testImplementation group: 'org.springframework', name: 'spring-test'
-
-    testImplementation group: 'net.bytebuddy', name: 'byte-buddy', version: versions.bytebuddy
-    testImplementation group: 'net.bytebuddy', name: 'byte-buddy-agent', version: versions.bytebuddy
-
-    testImplementation group: 'org.mockito', name: 'mockito-junit-jupiter', version: '4.4.0'
+    testImplementation 'com.github.hmcts:rd-commons-lib:0.1.0'
+    testImplementation group: 'org.mockito', name: 'mockito-junit-jupiter', version: '5.14.2'
 
     testImplementation group: 'org.powermock', name: 'powermock-api-mockito2', version: '2.0.9'
 
+    // tests on version 2 fails on flyway migrations, something like "incorrect syntax"
     testImplementation group: 'com.h2database', name: 'h2', version: '1.4.200'
 
-    testImplementation('com.github.tomakehurst:wiremock:2.35.2') {
-        exclude group: 'com.github.jknack'
-    }
+    testImplementation group: 'org.wiremock', name: 'wiremock', version: '3.10.0'
 
     testImplementation group: 'org.springframework.boot', name: 'spring-boot-starter-test', {
         exclude group: 'junit', module: 'junit'
         exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
     }
 
+    testImplementation group: 'org.codehaus.sonar-plugins', name: 'sonar-pitest-plugin', version: versions.sonarPitest
 
-    testImplementation group: 'org.pitest', name: 'pitest', version: versions.pitest
-    testImplementation group: 'info.solidsoft.gradle.pitest', name: 'gradle-pitest-plugin', version: versions.gradlePitest
-    testImplementation group: 'org.pitest', name: 'pitest-junit5-plugin', version: versions.junitPitest
-
-    testImplementation group: 'org.codehaus.sonar-plugins', name: 'sonar-pitest-plugin', {
-        exclude group: 'junit', module: 'junit'
-        exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
-    }, version: versions.sonarPitest
-
-    testImplementation 'io.github.openfeign:feign-jackson:12.5'
+    testImplementation 'io.github.openfeign:feign-jackson:13.5'
```

Comment:
implementation uses 13.3. either uplift it to 13.5 as well or else align, please